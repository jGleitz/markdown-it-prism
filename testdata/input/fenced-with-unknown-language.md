# Test

This is a fenced code block with language specification of an unknown language:

```this-is-not-a-language
public class Foo() {
	public Foo(bar) {
		System.out.println(bar);
	}
}
```
