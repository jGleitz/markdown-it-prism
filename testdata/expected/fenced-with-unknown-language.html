<h1>Test</h1>

<p>This is a fenced code block with language specification of an unknown language:</p>
<pre class="language-this-is-not-a-language"><code class="language-this-is-not-a-language">
	public class Foo() {
		public Foo(bar) {
			System.out.println(bar);
		}
	}
</code></pre>
